cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -739,13 +739,15 @@ def _two_qubit_clifford_matrices(
 
 
 def _random_single_q_clifford(
-    qubit: 'cirq.Qid', num_cfds: int, cfds: Sequence[Sequence['cirq.Gate']]
+    qubit: 'cirq.Qid', num_cfds: int, cfds: Sequence[Sequence['cirq.ops.SingleQubitCliffordGate']]
 ) -> List['cirq.Operation']:
     clifford_group_size = 24
-    operations = [[gate(qubit) for gate in gates] for gates in cfds]
+    operations = [[gate.to_phased_xz_gate()(qubit) for gate in gates] for gates in cfds]
     gate_ids = list(np.random.choice(clifford_group_size, num_cfds))
     adjoint = _reduce_gate_seq([gate for gate_id in gate_ids for gate in cfds[gate_id]]) ** -1
-    return [op for gate_id in gate_ids for op in operations[gate_id]] + [adjoint(qubit)]
+    return [op for gate_id in gate_ids for op in operations[gate_id]] + [
+        adjoint.to_phased_xz_gate()(qubit)
+    ]
 
 
 def _random_two_q_clifford(
```

Comment:
Do we need to do this for 2-qubit cliffords as well, or are those in the gateset?

Reply:
Sorry - this is obsolete. I already figured this out for myself :-)